M2/Macaulay2/packages/Msolve.m2

```diff
@@ -0,0 +1,585 @@
+newPackage(
+	"Msolve",
+	Version => "1.24.05", 
+    	Date => "July 2024",
+    	Authors => {{Name => "Martin Helmer", 
+		  Email => "mhelmer@ncsu.edu", 
+		  HomePage => "http://martin-helmer.com/"}, {Name => "Mike Stillman", 
+		  Email => "mike@math.cornell.edu", 
+		  HomePage => "https://math.cornell.edu/michael-e-stillman"},{Name => "Anton Leykin", 
+		  Email => "leykin@math.gatech.edu", 
+		  HomePage => "https://antonleykin.math.gatech.edu/"}},
+	  Keywords => {"Groebner Basis Algorithms" , "Interfaces"},
+    	Headline => "An interface to the msolve package (https://msolve.lip6.fr/) which computes Groebner Basis and does real root isolation",
+	PackageImports => { "Elimination", "Saturation" },
+    	AuxiliaryFiles => true,
+	DebuggingMode => false
+	)
+
+---------------------------------------------------------------------------
+
+export{
+    "msolveGB",
+    "msolveSaturate",
+    "msolveEliminate",
+    "msolveRUR",
+    "msolveLeadMonomials",
+    "msolveRealSolutions"
+    }
+
+importFrom_Core { "raw", "rawMatrixReadMsolveFile" }
+
+-- used in msolveEliminate
+importFrom_Core { "monoidIndices" }
+importFrom_Elimination { "eliminationRing" }
+
+---------------------------------------------------------------------------
+
+msolveMinimumVersion = "0.7.0"
+msolveProgram = findProgram("msolve", "msolve --help",
+    MinimumVersion => (msolveMinimumVersion, "msolve -V"),
+    RaiseError => false,
+    Verbose => debugLevel > 0)
+
+if msolveProgram === null then (
+    if not (options currentPackage).OptionalComponentsPresent
+    then ( printerr "warning: msolve cannot be found; ending"; end )
+    else ( printerr "warning: msolve found but its version is older than v" | msolveMinimumVersion;
```

Comment:
This line won't run, you need parentheses around the argument. (a bit late I know)